core/modules/user/user.install

```diff
@@ -1219,6 +1219,20 @@ function user_update_1017() {
   $config->save();
 }
 
+/**
+ * Set default values for core password constraints.
+ */
+function user_update_1018() {
+  $constraints = array(
+    'constraints_active' => FALSE,
+    'min_length' => 8,
+    'not_same_as_username' => TRUE,
+    'not_same_as_email' => TRUE,
+    'case_insensitive_comparison' => FALSE,
```

Comment:
Use `1`/`0` instead of `TRUE`/`FALSE`, plus I think that the default should be to use case insensitivity for password strength evaluation (which was the case before introducing this change):

```suggestion
    'constraints_active' => 0,
    'min_length' => 8,
    'not_same_as_username' => 1,
    'not_same_as_email' => 1,
    'case_insensitive_comparison' => 1,
```